export/distro/http_test.go

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright (c) 2018 Cavium
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package distro
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"strconv"
+	"testing"
+
+	"github.com/edgexfoundry/edgex-go/core/domain/models"
+)
+
+const ()
+
+func TestHttpSender(t *testing.T) {
+	const (
+		msgStr = "test message"
+		path   = "/somepath/foo"
+	)
+
+	var tests = []struct {
+		name string
+		addr models.Addressable
+	}{
+		{"noMethod", models.Addressable{
+			Protocol: "http",
+			Path:     path}},
+		{"get", models.Addressable{
+			Protocol:   "http",
+			HTTPMethod: models.MethodGet,
```

Comment:
@feclare I believe the build is failing due to the remaining models.MethodGet/Post and so on in this file. A later PR was merged before this one wherein all of the HTTP verbs were repointed. Once this edit is made, should be good to go.

Reply:
Thanks @tsconn23 , it should be fixed now.